client/src/clock-register/clock-register.controller.js

```diff
@@ -44,6 +44,13 @@
                 break;
               }
 
+              case 500: {
+                if (error.data.message) {
```

Comment:
I think this handling should be done in the first `if` on line 31. Also it looks like an inconsistency with `error.message`.